sdk/metricsadvisor/ai-metrics-advisor/src/metricsAdvisorKeyCredentialPolicy.ts

```diff
@@ -55,8 +112,8 @@ class MetricsAdvisorKeyCredentialPolicy extends BaseRequestPolicy {
     credential: MetricsAdvisorKeyCredential
   ) {
     super(nextPolicy, options);
-    this.subscriptionKey = credential.subscriptionKey;
-    this.apiKey = credential.apiKey;
+    this.subscriptionKey = credential.getSubscriptionKey();
```

Comment:
We might need to store the credential now, to support key rotation?  Like if a credential's keys are updated, we sign the request using the updated keys.

Reply:
How to do this part?